src/Renderer/PanoramicMesh.js

```diff
@@ -6,11 +6,10 @@
  */
 
 
-import NodeMesh from './NodeMesh';
-
+import * as THREE from 'three';
```

Comment:
Yeah, but the general style in itowns is to `import *` so let's keep it coherent (ie: we can do this but in a dedicated PR :) )